src/DotNetEnv/Parsers.cs

```diff
@@ -203,7 +203,7 @@ private static string ToUtf32Char (string hex)
                 .Or(Parse.WhiteSpace.AtLeastOnce().Text())
                 .AtLeastOnce()
                 .Select(strs => new ValueActual(strs))
-            ).Many().Select(vs => new ValueCalculator(vs));
+            ).Many().Select(vs => new ValueCalculator(vs.ToArray()));
```

Comment:
Why is this getting added?

Reply:
Willbe solved with reverting that: https://github.com/tonerdo/dotnet-env/pull/93/files/e1e5f17d1f16eb4761136bffe777cade0240bdd5#r1510010054